docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+volumes:
+  data:
+
+services:
+  postgres:
+    container_name: backend-postgres
+    image: postgres:14
+    environment:
+      - POSTGRES_USER=admin
+      - POSTGRES_PASSWORD=postgrespw
```

Comment:
should we worry about environment variables and not letting secrets be out? I don't think it's a problem now but might be once we deploy? 🤔